resources/views/projects/details.blade.php

```diff
@@ -0,0 +1,54 @@
+@extends('layouts.app')
+
+@section('content')
+
+<div class="padded content">
+	<div>
+		<!-- Project Name -->
+		<h1 class="ui left aligned container">{{ $project->name }}</h1>
+		<!-- Video Pitch -->
+		<div class="ui left aligned container">
+			<iframe width="560" height="315" src="{{ $project->video }}" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen style="display: block;margin: 5% auto;"></iframe>
```

Comment:
Let's test this on mobile. Creo que ponerle un width y height fijo en el style puede ocasionar que no se vea muy bien en mobile. 

Puedes modificarlo en el `sass` si prefieres y testearlo en el `responsive design mode` de tu browser.